apps/api/src/report/report.service.ts

```diff
@@ -1,14 +1,14 @@
 import { BadRequestException } from '@nestjs/common';
-import { getStartEndDateOfWeek, isValidDateFormat } from 'src/common/util';
+import { isValidDateFormat } from 'src/common/util';
 import { getManager } from 'typeorm';
 import dayjs from 'dayjs';
 
-const WEEKS_PER_PERIOD = 4;
-const PERIOD_START_DATE = '2022-05-01';
+const PERIOD_START_DATE = '2021-04-01';
```

Comment:
Erica mentioned that it should be 2022-05-01 in IEN-345. Is this for testing or is there an update from clients?